addons/SpellBook/readme.md

```diff
@@ -0,0 +1,20 @@
+# SpellBook
+
+This addon helps you find missing spells. You can search by job and level,
+or by category.
+
+## Usage
+
+```
+//spbk help -- show this help text
+//spbk [current] -- Shows learnable spells based on current main and sub job and level.
+//spbk <main|sub> [<level|all>] -- Show missing spells for current main or sub job. Defaults to the job's current level.
+//spbk <job> [<level|all>] -- Show missings spells for specified job and level. Defaults to the job's level.
+//spbk <category> [all]-- Show learnable spells by category. Limited to spells which are learnable, unless all is added after the category.
+Categories: whitemagic, blackmagic, songs, ninjustu, summoning, bluemagic, geomancy, trusts, all (Trusts are not included in all)
+Note: A limit on spent job points can be specified for spells learned from Gifts by entering a level of 100-1500. Spells are never given as Gifts for less than 100 jp.
+```
```

Comment:
A bit of a nitpick, but some consistency in the comments would be neat. Some sentences start with lowercase, some with uppercase. Some end with a period, some don't. Some start with a singular `shows`, others with a declarative form `show`. Might wanna tidy that up :)

Personally I'd probably make them singular code blocks for each command, with a description below, to match a more common format among addons, but that's just a preference,